docs/wallet_browser_extension/intro.md

```diff
@@ -0,0 +1,53 @@
+# ParallelChain Wallet Extension
+
+ParallelChain Wallet Extension is an extension for managing your XPLL and integrating your DApp.
+It allows your DApp to interact with your DApp users' XPLL accounts and make smart contract calls through the extension.
+
+# Features
+
+- Browser Extension
+  - Managing XPLL Wallet (Like in [Explorer Wallet](https://explorer.parallelchain.io/))
+    - Sending Transactions
+    - Staking
+  - Triggering Confirmation for smart contract or send token calls
```

Comment:
Inconsistent capitalisation in this section

Reply:
Solved by 0486a15.